### ISSUE TITLE
Formally convert SV info fields

### ISSUE THREAD

**Issue by fnothaft** [bigdatagenomics/adam#358]
#351 is caused because we don't fully handle the conversion of symbolic alleles. I've added a naïve workaround to #354 that fixes this by capturing the display string for the symbolic allele in the variant record; I think this is similar to what @arahuja did downstream in https://github.com/hammerlab/guacamole/pull/130? However, it'd be best to more fully implement the conversion. I've pushed a first crack at a schema for expressing SV's to https://github.com/bigdatagenomics/bdg-formats/pull/27 and would appreciate comments.

CCing @arahuja for comment.

**Comment by heuermh** [comment]
Sorry, hard to follow all the cross-linked issues, but it appears they're all closed/merged.  Can this be closed?

**Comment by fnothaft** [comment]
Yes, think so.
